TYPO3.Neos/Documentation/References/NodeMigrations.rst

```diff
@@ -0,0 +1,282 @@
+that are marked as removed and applies the ``RemoveNode`` transformation on them:
+
+.. code-block:: yaml
+
+  up:
+    comments: 'Delete removed nodes that were published to "live" workspace'
+    warnings: 'There is no way of reverting this migration since the nodes will be deleted in the database.'
+    migration:
+      -
+        filters:
+          -
+            type: 'IsRemoved'
+            settings: []
+          -
+            type: 'Workspace'
+            settings:
+              workspaceName: 'live'
+        transformations:
+          -
+            type: 'RemoveNode'
+            settings: []
+
+  down:
+    comments: 'No down migration available'
+
+Like all migrations the file should be placed in a package inside the ``Migrations/TYPO3CR`` folder where it will be picked
+up by the CLI tools provided with the content repository:
+
+- ``./flow node:migrationstatus``
+- ``./flow node:migrate``
+
+Use ``./flow help <command>`` to get detailed instructions. The migrationstatus command also prints a short description
+for each migration.
+
+Transformations Reference
+-------------------------
+
+AddDimensions
+~~~~~~~~~~~~~
+
+Add dimensions on a node. This adds to the existing dimensions, if you need to overwrite existing dimensions, use
+SetDimensions.
+
+Options Reference:
+
+``dimensionValues`` (array)
+  An array of dimension names and values to set.
+``addDefaultDimensionValues`` (boolean)
+  Whether to add the default dimension values for all dimensions that were not given.
+
+AddNewProperty
+~~~~~~~~~~~~~~
+
+Add the new property and its value.
+
+Options Reference:
+
+``newPropertyName`` (string)
+  The name of the new property to be added.
+``value`` (mixed)
+  Property value to be set.
+
+ChangeNodeType
+~~~~~~~~~~~~~~
+
+Change the node type.
+
+Options Reference:
+
+``newType`` (string)
+  The new Node Type to use as a string.
+
+ChangePropertyValue
+~~~~~~~~~~~~~~~~~~~
+
+Change the value of a given property.
+
+This can apply two transformations:
+- If newValue is set, the value will be set to this, with any occurrences of the ``currentValuePlaceholder`` replaced
+  with the current value of the property.
+- If search and replace are given, that replacement will be done on the value (after applying the ``newValue``, if set).
+
+Options Reference:
+
+``property`` (string)
+  The name of the property to change.
+``newValue`` (string)
+  New property value to be set.
+
+  The value of the option "currentValuePlaceholder" (defaults to "{current}") will be used to include the current
+  property value into the new value.
+``search`` (string)
+  Search string to replace in current property value.
+``replace`` (string)
+  Replacement for the search string.
+``currentValuePlaceholder`` (string)
+  The value of this option (defaults to ``{current}``) will be used to include the current property value into the new
+  value.
```

Comment:
this information is not enough for me to understand how currentValuePlaceholder work, maybe an example would help?